draft-ietf-tls-subcerts.md

```diff
@@ -347,7 +353,31 @@ advertised by the client in the "signature_algorithms" extension.  A delegated
 credential MUST NOT be negotiated otherwise, even if the client advertises
 support for delegated credentials.
 
-On receiving a delegated credential and a certificate chain, the client
+### Client authentication
+
+A server which supports this specification SHALL send an empty
+"delegated_credential" extension in the CertificateRequest message when
+asking for client authentication.  If the client receives a
+delegated credential without indicating support, then the client MUST abort with
+an "unexpected_message" alert.
+
+If the extension is present, the client MAY send a delegated credential; if the
+extension is not present, the client MUST NOT send a delegated credential.
+The client MUST ignore the extension unless TLS 1.3 or a later version is
+negotiated.
+
+The client MUST send the delegated credential as an extension in the
+CertificateEntry of its end-entity certificate; the server SHOULD ignore
+delegated credentials sent as extensions to any other certificate.
+
+The algorithm and expected_cert_verify_algorithm fields MUST be of a type
+advertised by the server in the "signature_algorithms" extension.  A delegated
+credential MUST NOT be negotiated otherwise, even if the server advertises
+support for delegated credentials.
```

Comment:
Should we specify the server behavior if it receives a DC indicating an algorithm it doesn't support? Presumably it would send an "illegal_parameter" alert, or something.